projects/balanced/pools.js

```diff
@@ -0,0 +1,128 @@
+module.exports = [
+    {
+        chainId: 1,
+        id: 1,
+        name: 'sICX/ICX',
+        baseCurrencyKey: 'sICX',
+        quoteCurrencyKey: 'ICX',
+    },
+    {
+        chainId: 1,
+        id: 2,
+        name: 'sICX/bnUSD',
+        baseCurrencyKey: 'sICX',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 3,
+        name: 'BALN/bnUSD',
+        baseCurrencyKey: 'BALN',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 4,
+        name: 'BALN/sICX',
+        baseCurrencyKey: 'BALN',
+        quoteCurrencyKey: 'sICX',
+    },
+    {
+        chainId: 1,
+        id: 5,
+        name: 'IUSDC/bnUSD',
+        baseCurrencyKey: 'IUSDC',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 15,
+        name: 'IUSDT/bnUSD',
+        baseCurrencyKey: 'IUSDT',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 10,
+        name: 'USDS/bnUSD',
+        baseCurrencyKey: 'USDS',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 7,
+        name: 'OMM/sICX',
+        baseCurrencyKey: 'OMM',
+        quoteCurrencyKey: 'sICX',
+    },
+    {
+        chainId: 1,
+        id: 6,
+        name: 'OMM/IUSDC',
+        baseCurrencyKey: 'OMM',
+        quoteCurrencyKey: 'IUSDC',
+    },
+    {
+        chainId: 1,
+        id: 8,
+        name: 'OMM/USDS',
+        baseCurrencyKey: 'OMM',
+        quoteCurrencyKey: 'USDS',
+    },
+    {
+        chainId: 1,
+        id: 9,
+        name: 'CFT/sICX',
+        baseCurrencyKey: 'CFT',
+        quoteCurrencyKey: 'sICX',
+    },
+    {
+        chainId: 1,
+        id: 17,
+        name: 'GBET/bnUSD',
+        baseCurrencyKey: 'GBET',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 31,
+        name: 'FIN/bnUSD',
+        baseCurrencyKey: 'FIN',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 48,
+        name: 'FRMD/sICX',
+        baseCurrencyKey: 'FRMD',
+        quoteCurrencyKey: 'sICX',
+    },
+    {
+        chainId: 1,
+        id: 50,
+        name: 'CROWN/bnUSD',
+        baseCurrencyKey: 'CROWN',
+        quoteCurrencyKey: 'bnUSD',
+    },
+    {
+        chainId: 1,
+        id: 50,
+        name: 'CLAW/sICX',
+        baseCurrencyKey: 'CLAW',
+        quoteCurrencyKey: 'sICX',
```

Comment:
CROWN/bnUSD and CLAW/sICX seems to have same id.

Reply:
Nice spotting, fixed, thanks!